packages/roosterjs-editor-plugins/lib/plugins/WatermarkV2/WatermarkV2.ts

```diff
@@ -0,0 +1,119 @@
+import {
+    EditorPlugin,
+    Entity,
+    IEditor,
+    PluginEvent,
+    PluginEventType,
+} from 'roosterjs-editor-types';
+import { kebabCase } from './utils';
+
+const ENTITY_TYPE = 'WATERMARK_WRAPPER';
+const WATERMARK_ATTR = 'data-rooster-watermark';
+const STYLE_ID = 'watermarkStyle';
+
+const DEFAULT_STYLE = {
+    color: 'rgba(0,0,0,0.3)',
+    position: 'absolute',
```

Comment:
So this means the editor DIV must have position either to be absolute or relative. But this is not always true. I need to think about if we can always add a position:relative if there is not.